src/libs/ajax/Metrics.ts

```diff
@@ -23,6 +23,7 @@ export const Metrics = (signal?: AbortSignal) => {
       event,
       properties: {
         ...details,
+        // Observation: The `true` and `false` sides of this ternary look like they may be reversed?  Perhaps someone who knows why it was done this way can comment explaining the logic here
```

Comment:
Better to add a comment that actually explains it I think. 😁 

```suggestion
        // Users who have not registered are considered anonymous users. Send an anonymized distinct_id in that case; otherwise the user identity is captured via the auth token.
```